pkg/skaffold/event/event.go

```diff
@@ -128,15 +129,16 @@ func (ev *eventHandler) forEachEvent(callback func(*proto.LogEntry) error) error
 	return <-listener.errors
 }
 
-func emptyState(build latest.BuildConfig) proto.State {
+func emptyState(c latest.Pipeline, kc string) proto.State {
```

Comment:
nit: these variables names seem strange. i'm actually not even sure what `kc` is here

Reply:
`kc` stands for kubecontext and `c` is Pipelin